### PR TITLE
New Language Translation (Slovak)

### PR DESCRIPTION
**Pull request by JustGejb**
# SCP:SL Translation PR Template

## Language
Slovak

## Type of change

- [x] Added new language translation

## Have you tested the changes you've made?
- [x] Yes

# Checklist:
- [Agree] I have made sure to not commit changes to other stuff than my own or the default english one
- [Agree] I have ran `EncodingNormalizer.exe` on my changed files before commiting 
   -  (Exception to this is when edits has __only__ been made in-browser)
- [Agree] I have made sure that I'm not submitting a duplicate translation 
- [Agree] I have not included any advertisements/pointers to non-game stuff other than author in the folder name

### CODE REVIEW

File: Translations/Slovak (Slovenčina)/WeaponManager.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Na spustenie pouzite lubovoln� klaves so sipkou.
```
Comment (MichalPetryka) [inline Translations/Slovak (Slovenčina)/WeaponManager.txt]: Missing character?

**Comment by JustGejb** [comment]
Oh god I'm so stupid thanks for all of it I'll fix it as soon as possible

**Comment by JustGejb** [comment]
PS: It is translated it's just that Czech and Slovak are very similiar

File: Translations/Slovak (Slovenčina)/Connection_Errors.txt [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+<size=80>ODPOJENÝ</size>\nZlé nastavenie servera (nekompatibilita verzií).
+<size=80>ODPOJENÝ</size>\nZlé nastavenie servera.
+<size=80>ODPOJENÝ</size>\nZadaná adresa nemôže byť dešifrované. Uistite sa, že sa jedná o správnu adresu.
+<size=80>ODPOJENÝ</size>\nObjavil sa neočakávaný problém!
+<size=80>Čakajte prosím</size>\n<size=35>Nadväzujem spojenie...</size>
+<size=80>Restart kola</size>\n<size=35>Server reštartuje kolo...</size>
+<size=80>Čakajte prosím</size>\n<size=35>Načítanie tutoriálu...</size>
+<size=80>NEKOMPATIBILITA VERZIÍ</size>\nVaša verzia hry je iná ako verzia hry na serveri\n(Vaša - [your]; Serveru - [server])
+<size=80>CONNECTED</size>
+<size=35>Loading the facility...</size>
```
Comment (MichalPetryka) [inline Translations/Slovak (Slovenčina)/Connection_Errors.txt]: And this?

Comment (JustGejb) [reply Translations/Slovak (Slovenčina)/Connection_Errors.txt]: Oooh sorry I didn't notice it I'll fix it

**Comment by JustGejb** [comment]
@MichalPetryka I see you requested changes but I don't see anything...

File: Translations/Slovak (Slovenčina)/Class_Descriptions.txt [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+Zabi všetky\nNenechaj kohokoľvek utiecť\nSpolupracuj s ostatnými <color=#f00>SCP</color>
+Pomôž <color=#FF8E00>personálu Triedy-D</color> utiecť\nNeutralizuj ostatné objekty
+Zabi všetky\nNenechaj kohokoľvek utiecť\nSpolupracuj s ostatnými <color=#f00>SCP</color>
+Zabi všetky\nNenechaj kohokoľvek utiecť\nSpolupracuj s ostatnými <color=#f00>SCP</color>
+Pomôž <color=#FFFF7CFF>vedcom</color> utiecť\nNeutralizuj ostatné objekty
+Pomôž <color=#FFFF7CFF>vedcom</color> utiecť\nNeutralizuj ostatné objekty
+Pomôž <color=#FFFF7CFF>vedcom</color> utiecť\nNeutralizuj ostatné objekty
+Pre dokončenie postupujte podľa pokynov.
+Help <color=#FFFF7CFF>scientists</color> to escape\nCooperate with <color=#0096FF>MTF</color>\nNeutralize other subjects
+Kill everyone.\nPrevent escape.\nCooperate with other <color=#f00>SCPs.</color>
```
Comment (MichalPetryka) [inline Translations/Slovak (Slovenčina)/Class_Descriptions.txt]: Not translated?

Comment (JustGejb) [reply Translations/Slovak (Slovenčina)/Class_Descriptions.txt]: On it :)

**Comment by MichalPetryka** [comment]
Ok, merging for now, any issues will be fixed in future

**Pull request merged**